modules/core/src/lib/deck.js

```diff
@@ -685,33 +703,33 @@ export default class Deck {
     }
   }
 
+  _onPointerDown(event) {
+    this._requestPick({
+      callback: null,
+      event,
+      mode: 'hover',
+      immediate: true
+    });
+  }
+
   _onPointerMove(event) {
```

Comment:
Could be worth checking if the mouse delta is less than a certain amount, so you only pick when  the mouse has settled on something.

Reply:
That would be a breaking change.